src/Home.tsx

```diff
@@ -48,6 +48,14 @@ const HomePage: React.FC = () => {
     return dayObject ? dayObject.full : '';
   };
 
+  const buttons = [
```

Comment:
maybe think of a meaningful name for "buttons"

Reply:
ok